### PR TITLE
Massive update of mappings

### PR DESCRIPTION
**Pull request by joebeernink**
Massive update of mappings

### CODE REVIEW

File: SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/OntologyMapper.Mapped.csproj [hunk 1]
```diff
@@ -8,8 +8,8 @@
 		<Version>$(VersionPrefix)</Version>
 		<Authors>Microsoft</Authors>
 		<Description>Support for converting from one Mapped Ontology to a different DTDL Based Ontology</Description>
-		<AssemblyVersion>0.9.1</AssemblyVersion>
-		<PackageVersion>0.9.1-preview</PackageVersion>
+		<AssemblyVersion>0.9.19</AssemblyVersion>
```
Comment (Tyler-Angell) [inline SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/OntologyMapper.Mapped.csproj:11]: Why such a large jump?

Comment (joebeernink) [reply SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/OntologyMapper.Mapped.csproj:11]: Have been testing locally. That's how many tries it took me to get it right

File: SmartPlaces.Facilities/lib/OntologyMapper.Mapped/test/OntologyMapper.Mapped.Test.csproj [hunk 2]
```diff
@@ -21,12 +21,13 @@
 		  <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
 		</PackageReference>
 		<PackageReference Include="Divergic.Logging.Xunit" Version="4.2.0" />
-		<PackageReference Include="Mapped.Ontologies.Core.Dtdl" Version="1.2.490" />
+		<PackageReference Include="Mapped.Ontologies.Core.Dtdl" Version="1.2.499" />
 		<PackageReference Include="Microsoft.ApplicationInsights" Version="2.21.0" />
 		<PackageReference Update="Microsoft.AspNet.WebApi.Client" Version="5.2.9" />
```
Comment (Tyler-Angell) [inline SmartPlaces.Facilities/lib/OntologyMapper.Mapped/test/OntologyMapper.Mapped.Test.csproj:26]: Shouldn't this be Include?

Comment (joebeernink) [reply SmartPlaces.Facilities/lib/OntologyMapper.Mapped/test/OntologyMapper.Mapped.Test.csproj:26]: I thought this was how we needed to set it for the newtonsoft issue?

File: SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json [hunk 3]
```diff
@@ -707,10 +707,10 @@
       "OutputDtmi": "dtmi:com:willowinc:connectedDataSourceId;1",
       "IsIgnored": true
     },
-    {
-      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Connected_Devices_Count_Status;1",
-      "OutputDtmi": "dtmi:com:willowinc:State;1"
-    },
+    //{
```
Comment (Tyler-Angell) [inline SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json:710]: Why is this Dtmi commented out?

File: SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json [hunk 4]
```diff
@@ -1200,13 +1200,13 @@
...
     {
-      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Duct_Static_Pressure_Setpoint;1",
+      "InputDtmi": "dtmi:mapped:core:Duct_Static_Pressure_Setpoint;1",
       "OutputDtmi": "dtmi:com:willowinc:StaticPressureSetpoint;1"
     },
-    {
-      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Econ_Cycle_Start_Stop_Status;1",
-      "OutputDtmi": "dtmi:com:willowinc:StartStopState;1"
-    },
+    //{
```
Comment (Tyler-Angell) [inline SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json:1206]: Why is this commented out?

File: SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json [hunk 5]
```diff
@@ -3430,23 +3429,23 @@
...
     {
-      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Real_Energy_Sensor;1",
+      "InputDtmi": "dtmi:mapped:core:Real_Energy_Sensor;1",
       "OutputDtmi": "dtmi:com:willowinc:EnergySensor;1"
     },
-    {
-      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Real_Power_Sensor;1",
-      "OutputDtmi": "dtmi:com:willowinc:PowerSensor;1"
-    },
+    //{
```
Comment (Tyler-Angell) [inline SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json:3435]: Why is this commented out?

File: SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json [hunk 6]
```diff
@@ -4503,16 +4506,226 @@
     {
       "InputDtmi": "dtmi:org:brickschema:schema:Brick:Zone_Unoccupied_Load_Shed_Command;1",
       "OutputDtmi": "dtmi:com:willowinc:UnoccupiedActuator;1"
+    },
+
+    // Random
```
Comment (Tyler-Angell) [inline SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json:4511]: In what way? They dont appear to be much different than the above

File: SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json [hunk 7]
```diff
@@ -4503,16 +4506,226 @@
...
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Wind_Direction_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:WindDirectionSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Zone_Air_Temperature_Setpoint;1",
+      "OutputDtmi": "dtmi:com:willowinc:ZoneAirTemperatureSetpoint;1"
+    },
+
+    // Mapped.Core
```
Comment (Tyler-Angell) [inline SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json:4669]: There's a whole bunch of Mapped.Core things further up, should those be moved here? Or these merged up there?

File: SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/BrickRec/mapped_v1_dtdlv2_Brick_1_3-REC_4_0.json [hunk 8]
```diff
@@ -222,355 +219,355 @@
...
       "OutputDtmi": "dtmi:org:w3id:rec:OfficeRoom;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Pump_Room;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Pump_Room;1",
       "OutputDtmi": "dtmi:org:w3id:rec:UtilitiesRoom;1"
     },
     {
-      "InputDtmi": "dtmi:mapped:core:Reception;1",
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Reception;1",
```
Comment (Tyler-Angell) [inline SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/BrickRec/mapped_v1_dtdlv2_Brick_1_3-REC_4_0.json:470]: Some of these mappings are surprising..  Reception isn't a Brick think according to the REC repo

**Review by alinamstanciu — APPROVED** [review]
LGTM

**Pull request merged**